src/sbt-test/sbt-scalafix/local-rules/build.sbt

```diff
@@ -13,12 +13,13 @@ val rules = project
   .disablePlugins(ScalafixPlugin)
   .settings(
     scalaVersion := Versions.scala212,
+    crossPaths := false,
```

Comment:
That does not seem to be enough, as I see `org.scala-lang:scala-library:2.13.1` in `service/Scalafix/externalDependencyClasspath`, which would probably cause all sorts of errors if the rule was not dummy. Investigating.

Reply:
Actually, I can't write a test to show this is a problem, so I am going to assume it's not. I can find 2 explanations for that (I am not sure about them though):
1. [scala-library is binary-compatible](https://docs.scala-lang.org/overviews/core/binary-compatibility-of-scala-releases.html#concretely)
1. the parent classloader already has the 2.12 library, so the 2.13 will effectively be ignored

Reply:
Somewhat unrelated to this PR, but looking at how class-loading works for this, I was surprised to see that https://github.com/scalacenter/sbt-scalafix/blob/v0.9.15-2/src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala#L22-L27 is not reused across `scalafix` invocations. Would it be possible to cache it via a private setting key maybe? I guess it would speed up scalafix when it's run on many projects in the same build.